docs: Clarify OUTPUT record vs output object in sync endpoints (#2689)
apify/apify-docs: sources/academy/tutorials/api/using_apify_from_php.md

## Getting the results from dataset

> Actors usually store their results in a default dataset. The [Actor runs endpoint](/api/v2/actor-runs) lets you get overall info about an Actor run's default dataset.

Reviewer: ```suggestion
An Actor usually stores its results in a default dataset created separately for each run. The [Actor runs endpoint](/api/v2/actor-runs) lets you get overall info about an Actor run's default dataset.
```

Author: Done in f2de4e3 — applied the suggestion.

---
_Generated by [Claude Code](https://claude.ai/code)_
